exo-go/src/application/tester.go

```diff
@@ -38,9 +38,12 @@ func NewTester(appConfig types.AppConfig, logger *Logger, appDir, homeDir string
 }
 
 // RunAppTests runs the tests for the entire application
-func (a *Tester) RunAppTests() error {
+func (a *Tester) RunAppTests(serviceName string) error {
 	a.logChannel <- fmt.Sprintf("Testing application %s", a.AppConfig.Name)
 	numFailed := 0
+	if serviceName != "" {
+		a.InternalServiceConfigs = map[string]types.ServiceConfig{serviceName: a.InternalServiceConfigs[serviceName]}
+	}
```

Comment:
I don't like overwriting `a.InternalServiceConfigs`. I think we should create another public function `RunServiceTests(serviceName string)`. We could have a helper which takes something of the same type of `a.InternalServiceConfigs` which we call either with `a.InternalServiceConfigs` or the custom object you build here when running a single service